src/components/EntityPage/Participant/Utils/Holder.js

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import {Container} from "../../../FileRepoSidebar/ui";
+import { Link } from 'react-router-dom';
+import styled from "react-emotion";
+import {withTheme} from "emotion-theming";
+
+//taken from secondaryNavMenu
+const MenuWrapper = styled('ul')`
+  ${({ theme }) => theme.secondaryNav}
+`;
+
+class Holder extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    //wrap the item in a one-element array if the item is not itself an array
+    if(!(props.children instanceof Array)) {
```

Comment:
Also, since the values of `this.tabIDs` and `this.children` are calculated from the `props`, they should be recalculated each time the props change.

Easiest way is to calculate them in the `render`, but if you need them outside of the render, you can use the lifecycle event https://reactjs.org/docs/react-component.html#static-getderivedstatefromprops , and end-up using the state instead of properties on `this`.